sequence_processing_pipeline/templates/nuqc_job.sh

```diff
@@ -109,107 +102,53 @@ function mux-runner () {
             --adapter_fasta {{knwn_adpt_path}} \
             --html {{html_path}}/${html_name} \
             --json {{json_path}}/${json_name} \
-            --out1 ${r1_filt} \
-            --out2 ${r2_filt}
+            --stdout > ${r1_filt}
 
         # multiplex and write adapter filtered data all at once
         cat ${r1_filt} | \
             sed -r "1~4s/^@(.*)/@${i}${delimiter}\1/" \
             >> ${seqs_r1} &
-        cat ${r2_filt} | \
-            sed -r "1~4s/^@(.*)/@${i}${delimiter}\1/" \
-            >> ${seqs_r2} &
         cat ${r1_filt} | \
             gzip -c > ${r1_adapter_only} &
-        cat ${r2_filt} | \
-            gzip -c > ${r2_adapter_only} &
         wait
```

Comment:
```suggestion

```

Reply:
@antgonza this appears related to the comment above.